docs/new_instrument.rst

```diff
@@ -430,74 +580,48 @@ present.
-
+generally be put in the `init` function of each instrument.
 
-.. _supported-data-templates:
+.. code:: Python
 
-Supported Data Templates
-========================
+    def init(self):
+        """Initializes the Instrument object with instrument specific values.
 
+        Runs once upon instantiation.
 
-NASA CDAWeb
------------
+        Parameters
+        ----------
+        inst : (pysat.Instrument)
+            Instrument class object
 
-A template for NASA CDAWeb pysat support is provided. Several of the routines within are intended to be used with functools.partial in the new instrument support code. When writing custom routines with a new instrument file download support would be added via
+        """
 
-.. code:: python
-
-   def download(.....)
-
-Using the CDAWeb template the equivalent action is
-
-.. code:: python
-
-   download = functools.partial(methods.nasa_cdaweb.download,
-                                supported_tags)
+        self.acknowledgements = acknowledgements_string
+        self.references = references_string
 
-where supported_tags is defined as dictated by the download function. See the routines for cnofs_vefi and cnofs_ivm for practical uses of the NASA CDAWeb support code.
-|br|
+        return
 
-.. automodule:: pysat.instruments.methods.nasa_cdaweb
-   :members:
 
-Madrigal
---------
+Supported Instrument Templates
+------------------------------
 
-A template for Madrigal pysat support is provided. Several of the routines within are intended to be used with functools.partial in the new instrument support code. When writing custom routines with a new instrument file download support would be added via
-
-.. code:: python
+Instrument templates may be found at ``pysat.instruments.templates``
+and supporting methods may be found at ``pysat.instruments.methods``.
 
-    def download(.....)
+General
+^^^^^^^
 
-Using the Madrigal template the equivalent action is
-
-.. code:: python
-
-     def download(date_array, tag='', sat_id='', data_path=None, user=None,
-                  password=None):
-         methods.madrigal.download(date_array, inst_code=str(madrigal_inst_code),
-                                   kindat=str(madrigal_tag[sat_id][tag]),
-                                   data_path=data_path, user=user,
-                                   password=password)
-
-See the routines for `dmsp_ivm` and `jro_isr` for practical uses of the Madrigal support code.
-
-Additionally, use of the methods.madrigal class should acknowledge the CEDAR rules of the road.  This can be done by Adding
-
-.. code:: python
+A general instrument template is included with pysat,
+``pysat.instruments.templates.template_instrument``,
+that has the full set
+of required and optional methods, and docstrings, that may be used as a
+starting point for adding a new instrument to pysat.
 
-     def init(self):
+Note that there are general supporting methods for adding an Instrument.
+See :ref:`rst_general_data_general` for more.
```

Comment:
doesn't work.

Reply:
Added section reference

Reply:
The link goes more to the API top than the general section within api.rst. As far as I can tell the reference is written and located as directed.